members/state_ri_gov_gina_raimondo.yaml

```diff
@@ -20,24 +20,26 @@ contact_form:
         - name: phoneAreaCode
           selector: "#area"
           value: $PHONE_AREA_CODE
+          required: true
         - name: phoneExchange
           selector: "#exchange"
           value: $PHONE_EXCHANGE_CODE
+          required: true
         - name: phoneNumber
           selector: "#phnum"
           value: $PHONE_SUBSCRIBER_NUMBER
+          required: true
         - name: zip5
           selector: "#zip"
           value: $ADDRESS_ZIP5
+          required: false
         - name: message
           selector: "#message"
           value: $MESSAGE
           required: true
         - name: recaptcha
-          captcha_selector: "#recaptcha_challenge_image"
-          value: $CAPTCHA_SOLUTION
           selector: "#recaptcha_response_field"
-          captcha_id_selector: "#recaptcha_challenge_field"
+          value: you shall pass
```

Comment:
![shan't](https://m.popkey.co/5829be/VWKka.gif "Shall Not")

Reply:
![none shall pass](https://cdn.shopify.com/s/files/1/0436/7045/products/mens_t_shirt_-_monty_python_s_the_holy_grail_-_the_black_knight_none_shall_pass_-_white_cropped_300x300.jpg)